tests/regression/02-base/50-unknown_func_array.c

```diff
@@ -8,12 +8,7 @@ typedef struct arr {
     int *ptrs[LENGTH];
 } arr_t;
 
-// int mutate_array(arr_t a){
-//     int t = rand();
-//     for(int i=0; i<LENGTH; i++) {
-//         *(a.ptrs[i]) = t;
-//     }
-// }
```

Comment:
Maybe leave this commented out code here, such that one can understand why this invalidation is needed?